src/threading.c

```diff
@@ -49,6 +49,21 @@ JL_DLLEXPORT _Atomic(uint8_t) jl_measure_compile_time_enabled = 0;
 JL_DLLEXPORT _Atomic(uint64_t) jl_cumulative_compile_time = 0;
 JL_DLLEXPORT _Atomic(uint64_t) jl_cumulative_recompile_time = 0;
 
+JL_DLLEXPORT uint64_t jl_thread_up_time(void)
+{
+    jl_ptls_t ptls = jl_current_task->ptls;
+    return jl_record_time_for_tls_metric() - ptls->timing_tls.start_time;
+}
```

Comment:
IMO even though none of the other functions in this file have comments, any new functions we add should have top-level function comments.

Reply:
https://github.com/RelationalAI/julia/pull/181#discussion_r1781499757